src/models/ServerStyleSheet.js

```diff
@@ -139,6 +151,44 @@ export default class ServerStyleSheet {
     return this.instance.toReactElements()
   }
 
+  interleaveWithNodeStream(readableStream: stream.Readable) {
+    if (__SERVER__) {
+      const ourStream = new stream.Readable()
+
+      // $FlowFixMe
+      ourStream._read = () => {}
+
+      this.isStreaming = true
+      this.lastIndex = 0
+
+      readableStream.on('data', chunk => {
+        ourStream.push(
+          this.instance.tags
+            .slice(this.lastIndex)
+            .map(tag => tag.toHTML())
+            .join('') + chunk
+        )
+
+        this.lastIndex = this.instance.tags.length - 1
+      })
+
+      readableStream.on('end', () => {
+        this.closed = true
+        ourStream.push(null)
+      })
+
+      readableStream.on('error', err => {
+        ourStream.emit('error', err)
+      })
+
+      return ourStream
+    } else {
+      throw new Error(
+        'streaming is not implemented in browser builds of styled-components.'
```

Comment:
This should be something like "Streams only work in Node.js, not the browser. Please don't use xyz method in the browser.", since it won't ever be implemented in browser builds of styled-components